### DISCUSSION TITLE
New release - date

### DISCUSSION THREAD

**Discussion by aa89113** [GAA-UAM/scikit-fda#633] [Q&A]
Hello, I am looking forward to have the following fix:
[scipy-fix](https://github.com/GAA-UAM/scikit-fda/commit/2f1fb20e1bf705aa1ed6e95250d5f0fed57260e9)

Would you have an expected date for the next release?
Thank you
Marcio

**Comment by vnmabus** [comment]
So, normally I try to merge pending PRs and release a new version in summer, but this year I did not have enough bandwidth to do so, and I still have PRs pending to merge. I will try to finish them as soon as I can and do a release.

In the meantime remember that it is possible to install the current develop version if you need something not yet released:
```
python -m pip install git+https://github.com/GAA-UAM/scikit-fda.git@develop
```

**Reply by aa89113** [reply]
hi @vnmabus,
Many thanks for your reply.  I would prefer to have a pinned version, though ...
Would be possible to have a `0.9.2` version with the current develop version?

Thank you
Marcio

**Reply by vnmabus** [reply]
I did a release (version 0.10.1). Sorry for taking so long with this.
